styles/stylesheet.css

```diff
@@ -0,0 +1,378 @@
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+h2 {
+    margin-top: 3%;
+}
+
+
+
+h2,
+h3,
+h4,
+.dogs img,
+.dogs p,
+.all-dogs p {
+    margin-bottom: 5%;
+}
+
+.blog1 h2,
+.blog2 h2,
+.blog3 h2,
+.blog1 p,
+.blog2 p,
+.blog3 p {
+    margin-top: 0;
+    margin-bottom: 2%;
+}
+
+/* .blog1 p, .blog2 p , .blog3 p{
+    font-size: .9rem;
+} */
+
+h3,
+h4,
+.dogs p,
+.all-dogs p,
+.dogs button {
+    margin-left: 3%;
+    margin-right: 10%;
+}
+
+header {
+    background-color: black;
+    color: white;
+    background-image: url("../images/header-image.jpg");
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.logo {
+    font-weight: bold;
+    /* margin-left: 2.5%; */
+}
+
+.cart {
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: center;
+    font-size: .8rem;
+}
+
+header h1 {
+    color: #008A00;
+    font-style: italic;
+    font-size: 1.25rem;
+    /* margin-left: 2.5%; */
+}
+
+.hdr-left {
+    margin-left: 10%;
+    margin-top: 2%;
+    margin-bottom: 5%;
+    /* width:50%; */
+}
+
+.hdr-right {
+    margin-right: 10%;
+    margin-top: 5%;
+    margin-bottom: 5%;
+    /* width:5%; */
+}
+
+header button {
+    padding: 5% 40%;
+    background-color: #008A00;
+    border: 1px solid rgb(48, 44, 44);
+    color: white;
+    font-weight: bold;
+    font-size: .7rem;
+}
+
+header p {
+    font-size: .7rem;
+    padding-top: 30%;
+}
+
+header img,
+header p,
+header button {
+    margin-top: 10%;
+}
+
+.cart img {
+    width: 50%;
+    margin-right: 15%;
+}
+
+nav {
+    background-color: lightgray;
+}
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-content: flex-start;
+    flex-wrap: wrap;
+    padding-left: 10%;
+}
+
+nav li,
+footer li,
+nav p {
+    list-style: none;
+    padding: 1% 0% 1% 1%;
+    font-size: .75rem;
+    font-weight: bold;
+}
+
+nav a,
+footer a {
+    text-decoration: none;
+}
+
+
+#wrapper {
+    margin-left: 10%;
+    /* margin-right: 5%; */
+    /* background-color: lightblue; */
+}
+
+
+
+.intro {
+    display: grid;
+    grid-template-columns: 40% 40%;
+    grid-template-rows: auto auto;
+    margin-top: 5%;
+    margin-bottom: 5%;
+}
+
+.dogs,
+.all-dogs {
+    display: grid;
+    grid-template-columns: 25% 25% 25% 25%;
+    grid-template-rows: auto auto;
+    margin-top: 5%;
+    margin-bottom: 15%;
+    grid-gap: 3% 3%;
+
+}
+
+.dogs img {
+    width: 100%;
+}
+
+.dogs button,
+.all-dogs button {
+    width: 90%;
+    padding: 1% 10% 1% 10%;
+    background-color: #008A00;
+    color: white;
+    font-weight: bold;
+}
+
+.dogs img,
+.all-dogs img {
+    width: 100%;
+}
+
+.more-dogs {
+    width: 200px;
+    background-color: lightgray;
+    padding: 5% 5%;
+    padding-bottom: 25%;
+}
+
+.more-dogs a {
+    text-decoration: none;
+    padding: 0 15%;
+    width: 100%;
+
+    color: rgb(20, 66, 109);
+    font-weight: bold;
+
+}
+
+.position-button {
```

Comment:
This block of HTML is a little nested heavy. Just commenting here on the CSS. 

See if you can't eliminate the position relative and the nested stuff and just use margin to center things. I might better explain in person. When we get to Q&A part of class I might be able to talk through it. Just ask me, as I'll probably forget.